decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -1,488 +0,0 @@
-  # take over user accounts.
-  #
-  # config.enable_html_header_snippets = Decidim::Env.new("DECIDIM_ENABLE_HTML_HEADER_SNIPPETS").present?
-
-  # Allow organizations admins to track newsletter links.
-  # unless Decidim::Env.new("DECIDIM_TRACK_NEWSLETTER_LINKS", "auto").default_or_present_if_exists.to_s == "auto"
-  #   config.track_newsletter_links = Decidim::Env.new("DECIDIM_FORCE_SSL", "auto").default_or_present_if_exists
-  # end
-
-  # Amount of time that the download your data files will be available in the server.
-  # config.download_your_data_expiry_time = Decidim::Env.new("DECIDIM_DOWNLOAD_YOUR_DATA_EXPIRY_TIME", "7").to_i.days
-
-  # Max requests in a time period to prevent DoS attacks. Only applied on production.
-  # config.throttling_max_requests = Decidim::Env.new("DECIDIM_THROTTLING_MAX_REQUESTS", "100").to_i
-
-  # Time window in which the throttling is applied.
-  # config.throttling_period = Decidim::Env.new("DECIDIM_THROTTLING_PERIOD", "1").to_i.minutes
-
-  # Time window were users can access the website even if their email is not confirmed.
-  # config.unconfirmed_access_for = Decidim::Env.new("DECIDIM_UNCONFIRMED_ACCESS_FOR", "0").to_i.days
-
-  # A base path for the uploads. If set, make sure it ends in a slash.
-  # Uploads will be set to `<base_path>/uploads/`. This can be useful if you
-  # want to use the same uploads place for both staging and production
-  # environments, but in different folders.
-  #
-  # If not set, it will be ignored.
-  # config.base_uploads_path = Decidim::Env.new("DECIDIM_BASE_UPLOADS_PATH").to_s if Decidim::Env.new("DECIDIM_BASE_UPLOADS_PATH").present?
-
-  # SMS gateway configuration
-  #
-  # If you want to verify your users by sending a verification code via
-  # SMS you need to provide a SMS gateway service class.
-  #
-  # An example class would be something like:
-  #
-  # class MySMSGatewayService
-  #   attr_reader :mobile_phone_number, :code
-  #
-  #   def initialize(mobile_phone_number, code)
-  #     @mobile_phone_number = mobile_phone_number
-  #     @code = code
-  #   end
-  #
-  #   def deliver_code
-  #     # Actual code to deliver the code
-  #     true
-  #   end
-  # end
-  #
-  # config.sms_gateway_service = "MySMSGatewayService"
-
-  # Timestamp service configuration
-  #
-  # Provide a class to generate a timestamp for a document. The instances of
-  # this class are initialized with a hash containing the :document key with
-  # the document to be timestamped as value. The instances respond to a
-  # timestamp public method with the timestamp
-  #
-  # An example class would be something like:
-  #
-  # class MyTimestampService
-  #   attr_accessor :document
-  #
-  #   def initialize(args = {})
-  #     @document = args.fetch(:document)
-  #   end
-  #
-  #   def timestamp
-  #     # Code to generate timestamp
-  #     "My timestamp"
-  #   end
-  # end
-  #
-  #
-  # config.timestamp_service = "MyTimestampService"
-
-  # PDF signature service configuration
```

Comment:
This should be moved to a page like https://docs.decidim.org/en/v0.30/services/sms (i.e. https://docs.decidim.org/en/v0.30/services/pdf_signature)